app/jobs/rest_of_world_zones/rest_of_world_zone_update_job.rb

```diff
@@ -0,0 +1,55 @@
+module RestOfWorldZones
+  class RestOfWorldZoneUpdateJob < ApplicationJob
+    def perform(spree_member, rest_of_world_country_zone, profile_id)
+      @spree_member = spree_member
+      @rest_of_world_country_zone = rest_of_world_country_zone
+      @profile_id = profile_id
+      update_rest_of_world_zones
+    end
+
+    def update_rest_of_world_zones
+      @spree_member.class.name == 'Spree::Country' ? update_rest_of_world_country_zone : update_rest_of_world_state_zone
+    end
+
+    def update_rest_of_world_country_zone
```

Comment:
`destroy_rest_of_world_country_zone`?

Reply:
no, it just destroys the member, not whole zone